src/olympia/abuse/cinder.py

```diff
@@ -98,6 +92,28 @@ def get_context(self):
         }
 
 
+class CinderNonFxaUser(Cinder):
+    type = 'amo_unauthenticated_reporter'
+
+    def __init__(self, name, email):
+        self.name = name
+        self.email = email
+
+    def get_attributes(self):
+        return {
+            'id': f'{self.name} : {self.email}',
```

Comment:
btw, I've queried if this is a good id for a non-authenticated user in the cinder channel.  Waiting for a response on the question, but it's a minor detail.